src/config.js

```diff
@@ -7,6 +7,8 @@ export default {
       storagefacility: {
         searchUrl: (term, mid) =>
           `/api/storagefacility/v1/${mid.getPath()}/storagenodes/search?searchStr=${encodeURIComponent(term)}&`,
+        scanUrl: (uuid, mid) =>
```

Comment:
You might as well add support for a second `scanUrl` here.

`/api/storagefacility/v1/${mid.getPath()}/storagenodes/scan?oldBarcode=${oldBarcode}`

Where `oldBarcode` is a (most likely unique) numeric identifier for the old barcode labels.